Polyhedron/demo/Polyhedron/Plugins/Mesh_3/raw_image.ui

```diff
@@ -65,161 +84,191 @@
+          <property name="toolTip">
+           <string>Number of voxels along the Z axis.</string>
+          </property>
+          <property name="prefix">
            <string>z: </string>
           </property>
-          <property name="maximum" >
+          <property name="maximum">
            <number>100000000</number>
           </property>
          </widget>
         </item>
-        <item row="2" column="1" >
-         <widget class="QDoubleSpinBox" name="spacing_x" >
-          <property name="prefix" >
+        <item row="2" column="1">
+         <widget class="QDoubleSpinBox" name="spacing_x">
+          <property name="toolTip">
+           <string>Voxel size in x.</string>
+          </property>
+          <property name="prefix">
            <string>vx: </string>
           </property>
-          <property name="decimals" >
+          <property name="decimals">
            <number>5</number>
           </property>
-          <property name="value" >
+          <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
-        <item row="2" column="2" >
-         <widget class="QDoubleSpinBox" name="spacing_y" >
-          <property name="prefix" >
+        <item row="2" column="2">
+         <widget class="QDoubleSpinBox" name="spacing_y">
+          <property name="toolTip">
+           <string>Voxel size in y.</string>
+          </property>
+          <property name="prefix">
            <string>vy: </string>
           </property>
-          <property name="decimals" >
+          <property name="decimals">
            <number>5</number>
           </property>
-          <property name="value" >
+          <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
-        <item row="2" column="3" >
-         <widget class="QDoubleSpinBox" name="spacing_z" >
-          <property name="prefix" >
+        <item row="2" column="3">
+         <widget class="QDoubleSpinBox" name="spacing_z">
+          <property name="toolTip">
+           <string>Voxel size in z.</string>
+          </property>
+          <property name="prefix">
            <string>vz: </string>
           </property>
-          <property name="decimals" >
+          <property name="decimals">
            <number>5</number>
           </property>
-          <property name="value" >
+          <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
-        <item row="3" column="0" >
-         <widget class="QLabel" name="offset_label" >
-          <property name="text" >
+        <item row="3" column="0">
+         <widget class="QLabel" name="offset_label">
+          <property name="toolTip">
+           <string>Specifies how many bytes must be skipped before reading the data. (Use it to skip unhandled headers fo example)</string>
```

Comment:
typo: "fo example"

and maybe add a `.` at the end of the sentence inside the parenthesis.